nncf/torch/quantization/layers.py

```diff
@@ -52,6 +52,45 @@ def from_str(config_value: str) -> 'HWConfigType':
         raise RuntimeError("Unknown quantizer ONNX export mode string")
 
 
+class PTQuantizerConfig(QuantizerConfig):
```

Comment:
It would be a third class to describe parameters of quantization.

Please consider using `PTQuantizerSpec`

Reply:
Actually `PTQuantizerConfig` can be replaced by `PTQuantizerSpec` but then `PTQuantizerSpec` has to implement `QuantizerConfig` methods: `is_a_bitwidth_variant` and `compatible_with_a_unified_scale_linked_qconfig`. This would lead to large `PTQuantizerSpec` that could be used as a `QuantizerConfig` (and may be inherited from such).
Also `QuantizerConfig.is_a_bitwidth_variant` and `QuantizerConfig.compatible_with_a_unified_scale_linked_qconfig` are used only in PT case so they could be removed from `QuantizerConfig` and be placed in `PTQuantizerSpec`
In total `PTQuantizerSpec` would additionally have:

- `per_channel `attr
- `is_a_bitwidth_variant` method (this method will be removed from `QuantizerConfig`)
- `compatible_with_a_unified_scale_linked_qconfig` method (this method will be removed from `QuantizerConfig`)
- `get_state` method

`per_channel` attr is needed for `is_a_bitwidth_variant` and `compatible_with_a_unified_scale_linked_qconfig` methods
`get_state` is needed for saving to compression_state

@vshampor Do you think this kind of changes to `PTQuantizerSpec` and `QuantizerConfig` would be appropriate?